content/docs/create-a-new-react-app.md

```diff
@@ -8,87 +8,87 @@ prev: add-react-to-a-website.html
 next: cdn-links.html
 ---
 
-Use an integrated toolchain for the best user and developer experience.
+Gunakan *toolchains* yang terintegrasi untuk pengalaman pengguna dan pengembang yang terbaik.
 
-This page describes a few popular React toolchains which help with tasks like:
+Halaman ini menjelaskan beberapa *toolchains* yang bisa membantu menyelesaikan tugas seperti:
 
-* Scaling to many files and components.
-* Using third-party libraries from npm.
-* Detecting common mistakes early.
-* Live-editing CSS and JS in development.
-* Optimizing the output for production.
+* Penskalaan ke banyak *file* dan komponen.
+* Penggunaan *library* pihak ketiga dari *npm*.
+* Mendeteksi kesalahan umum lebih awal.
+* Pengeditan *CSS* dan *JS* secara langsung dalam tahap pengembangan.
+* Mengoptimalkan keluaran untuk tahap produksi.
 
-The toolchains recommended on this page **don't require configuration to get started**.
+*Toolchains* yang direkomendasikan pada halaman ini **tidak membutuhkan konfigurasi untuk memulai**.
 
-## You Might Not Need a Toolchain {#you-might-not-need-a-toolchain}
+## Anda mungkin tidak butuh *Toolchains* {#you-might-not-need-a-toolchain}
 
-If you don't experience the problems described above or don't feel comfortable using JavaScript tools yet, consider [adding React as a plain `<script>` tag on an HTML page](/docs/add-react-to-a-website.html), optionally [with JSX](/docs/add-react-to-a-website.html#optional-try-react-with-jsx).
+Jika anda tidak mengalami masalah yang dijelaskan di atas atau belum merasa nyaman menggunakan *tools* dari *JavaScript*, pertimbangkan untuk [menambahkan *React* sebagai *tag* `<script>` biasa pada halaman *HTML*](/docs/add-react-to-a-website.html). Anda dapat melakukan itu [dengan *JSX*](/docs/add-react-to-a-website.html#optional-try-react-with-jsx).
```

Comment:
anda in kata sapaan should be capitalized, for example:

- `Anda mungkin akan mengalami masalah` -> Kata sapaan
- `... untuk memperbaiki komponen anda` -> kepemilikan

😄

Reply:
JavaScript doesn't have to be italicized since it is a name 😄